### PR TITLE
Implement hooks: Milestone 4

### PR DESCRIPTION
**Pull request by pasytchangwa**
In this milestone, the following was done:

- Refactored the Calculator component (components/Calculator.js) from a class based component to a functional component with hooks. 
- Tested the app in the browser to assure that it keeps the same functionality.
- Fix README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,8 +13,11 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 The aim of this project is to learn how to use React.
 
-![Screenshot from 2021-08-18 18-16-06](https://user-images.githubusercontent.com/64914462/129924752-757fb6dd-8724-488b-887d-552e63e58f28.png)
+# Project online link
+
+[Click here](https://pasytchangwa.github.io/Math/) to see a sample demo.
```
Comment (Dipeshtwis) [inline README.md:18]: - [ ] Currently, the live link pointing to the README doesn't seem professional, please fix it.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @pasytchangwa, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Refactored the Calculator component (components/Calculator.js) from a class-based component to a functional component with hooks.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (pasytchangwa) [reply README.md:18]: Thank you for the review. I will look into it and see how to make it better.

Comment (pasytchangwa) [reply README.md:18]: fixed!

File: src/App.js [hunk 2]
```diff
@@ -4,11 +4,6 @@ import CalcApp from './components/Calculator';
 import './components/style.css';
 
 export default class App extends Component {
```
Comment (ZawadiSumba66) [inline src/App.js:6]: [OTIONAL]
- This might have been missed by the previous reviewer but kindly refactor your App.js file to use a functional component.

**Review by ZawadiSumba66 — APPROVED** [review]
Hi @pasytchangwa 👋 , 
## STATUS: APPROVED 👍 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**